Automation/Web/Get started with Playwright for Python.md

```diff
@@ -0,0 +1,69 @@
+## What is Playwright?
+
+Playwright is a test automation tool for Web testing. It supports testing on various browsers in both headed and headless mode. 
+You can write tests in JavaScript / TypeScript, C#, Java and Python. Depending on your language of choice, the setup slightly differs.
+
+This is a short and simple guide to quickly get started with Playwright for Python.
+
+
+## Requirements
+
+* Python 3.7 or newer.
+* Check [Playwright troubleshooting](https://playwright.dev/python/docs/troubleshooting) for more details.
+
+
+## Setup
+
+1. Create a new PyCharm project 
+   * Create and activate a [virtual environment](https://infinum.com/handbook/qa/automation/python/virtual-environment)
+2. Install [pytest-playwright](https://pypi.org/project/pytest-playwright/) plugin 
```

Comment:
Can we remove slashes `/` from the end of URLs? 
It looks prettier when looking at the code 😄

Reply:
Yep, not needed. Removed.